e2e/tests/ui-driven/src/mocks/geospatialMocks.ts

```diff
@@ -1,29 +1,35 @@
 import { OptionWithDataValues } from "../helpers/types";
+import { Feature, Polygon, Position } from "geojson";
+
+type ChangeHandlerProperties = {
+  label: string;
+  "area.squareMetres": number;
+  "area.hectares": number;
+};
+
+export type GeoJsonChangeHandler = Feature<Polygon, ChangeHandlerProperties>;
 
 export const mockPropertyTypeOptions: OptionWithDataValues[] = [
   { optionText: "Residential", dataValue: "residential" },
   { optionText: "Commercial", dataValue: "commercial" },
 ];
 
-import { Feature } from "geojson";
+const mockCoordinates: Position[][][] = [
+  [
+    [
+      [-0.633498, 51.605485],
+      [-0.633455, 51.605606],
+      [-0.633788, 51.605643],
+      [-0.634429, 51.605799],
+      [-0.634429, 51.605767],
+      [-0.633498, 51.605485],
+    ],
+  ],
+];
 
 export const mockMapGeoJson: Feature = {
```

Comment:
nit: `mockTitleBoundaryGeoJson` would be a more meaningful name here, everything is GeoJSON with the map !